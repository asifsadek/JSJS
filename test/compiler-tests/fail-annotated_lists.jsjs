val xs1: list string = ["1","2","3","4","5","6","7"];

val xs2: list list string = [[1,2,3],[4,5,6],[7,8,9],[10]];
