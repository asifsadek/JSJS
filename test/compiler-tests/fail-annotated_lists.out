Type error: expected value of type 'list list string', got a value of type 'list list num' instead
